.travis.yml

```diff
@@ -22,20 +22,22 @@ matrix:
         - ENABLE_PACKAGING=1
         - BUILD_SHARED_LIBS=ON
         - USE_VTK=1
-        - BUILD_DOCUMENTATION=1
+        - BUILD_DOCUMENTATION=OFF
```

Comment:
you also have to change into before_install.sh:
>     if [[ "$BUILD_DOCUMENTATION" == "1" ]]; then
        brew install Doxygen Graphviz # For building documentation

because else travis is not happy